cluster-agent/controllers/managed-gitops/eventloop/operation_event_loop.go

```diff
@@ -915,9 +926,38 @@ func deleteArgoCDApplicationOfDeletedApplicationRow(ctx context.Context, dbAppli
 		log.Error(firstDeletionErr, "Deletion of at least one Argo CD application failed.", "firstError", firstDeletionErr)
 		return shouldRetryTrue, firstDeletionErr
 	}
-
 	// success
 	return shouldRetryFalse, nil
+}
+
+func processOperation_GitOpsEngineInstance(ctx context.Context, dbOperation db.Operation, crOperation operation.Operation, opConfig operationConfig) (bool, error) {
+
+	if dbOperation.Resource_id == "" {
+		return shouldRetryTrue, fmt.Errorf("resource id was nil while processing operation: " + crOperation.Name)
+	}
+
+	dbGitopsEngineInstance := &db.GitopsEngineInstance{
+		Gitopsengineinstance_id: dbOperation.Instance_id,
+	}
+
+	log := opConfig.log.WithValues("Gitopsengineinstance_id", dbGitopsEngineInstance.Gitopsengineinstance_id)
+
+	err := opConfig.dbQueries.GetGitopsEngineInstanceById(ctx, dbGitopsEngineInstance)
+	if err != nil {
+		log.Error(err, "Unable to retrieve database GitopsEngineInstance row from database")
+		return shouldRetryTrue, err
+	} else {
+		// The ArgoCD CR name is given the name of the namespace it is being created in
+		fmt.Println(dbGitopsEngineInstance.Namespace_name)
```

Comment:
A few `fmt.Println` snuck into the code, and should be removed, like this one. If you look through the PR tab and search for `fmt.Println` you can find them all.